tools/ka-customizer-installer

```diff
@@ -3,6 +3,9 @@
 APPDIR=$HOME/.local/share/applications
 BINDIR=$HOME/.local/bin
 
+if ! flatpak info --show-commit com.endlessm.Sdk eos3.1; then
+    flatpak install eos-runtimes com.endlessm.Sdk eos3.1
+fi
```

Comment:
Can check for Platform sure. This is meant to be run on non converted eos images by design where it should already be installed, but can't hurt to check.

I can add a variable here, though honestly I'm not sure what this looks like come 3.2. Do we need to make sure all of 3.1, 3.2 and master are installed? Will some apps still be targeting the 3.1 runtime? Kinda suspect so, but I'm not totally worried about future proofing this quite yet.